Spring one

asaikali opened the pull request.

LGDoor committed:
Add AI backend and sample data. (#1)

- Added a ChatGPT-like REST API which can chat with the user and answer the questions about the products.
- Added brand new bikes and accessories with detailed specs and images.

asaikali committed:
Merge branch 'Azure-Samples:Azure' into SpringOne

LGDoor committed:
Add 25 sample data and images for accessories. (#2)

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

LGDoor committed:
Add frontend for AI (#5)

Co-authored-by: Zhe Li <lizhe@lizhe.org>
Co-authored-by: Zhe Li <zlhe@microsoft.com>
Co-authored-by: Zhe Li <Sneezry@users.noreply.github.com>

LGDoor committed:
Add pre-processing. (#6)

* Add code for preprocessing.

* Update the doc for preprocessing.

* add license for the images.

* Fix the data location.

---------

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

LGDoor committed:
Update readme.md

Sneezry committed:
add ai into home and cart

Sneezry committed:
Merge pull request #7 from selvasingh/zlhe-test

add ai into home and cart

LGDoor committed:
Tune the prompts.

Sneezry committed:
update api

LGDoor committed:
Rename the module and refactor code and API. (#8)

* merge: refactor code and API

update scripts

update greeting message

Update FAQ and images.

* Reformat and add comments.

---------

Co-authored-by: Dingmeng Xue <dixue@microsoft.com>
Co-authored-by: Raphael Yu <xiangy@microsoft.com>

Sneezry committed:
update ui

Sneezry committed:
update ui

Sneezry committed:
bug fix

Sneezry committed:
bugfix

Sneezry committed:
update frontend

Sneezry committed:
Merge pull request #9 from selvasingh/zlhe/newui

Update frontend

LGDoor committed:
Add logging for execution time.

Sneezry committed:
wording fix

Sneezry committed:
Merge pull request #11 from selvasingh/zlhe/wordingfix

wording fix

Sneezry committed:
move ai chat to nav bar

Sneezry committed:
Merge pull request #12 from selvasingh/zlhe/aichat

move ai chat to nav bar

Sneezry committed:
fix marked lib

Sneezry committed:
Merge pull request #13 from selvasingh/zlhe/aichat

fix marked lib

Sneezry committed:
fix marked lib

Sneezry committed:
fix marked lib

Sneezry committed:
fix marked lib

Sneezry committed:
scroll top

Sneezry committed:
Merge pull request #14 from selvasingh/zlhe/aichat

Update UI

LGDoor committed:
Change to gpt-35-turbo-16k and provide more context. (#15)

* Change to gpt-35-turbo-16k and provide more context.

* fix README.

---------

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

dingmeng-xue committed:
update faq and fix minor issue

dingmeng-xue committed:
Update FAQ and fix minor issue

Update FAQ and fix minor issue

markpollack committed:
adding spring ai project, updates (#17)

LGDoor committed:
Some minor fixes.

dingmeng-xue committed:
Remove lombok

dingmeng-xue committed:
Merge remote-tracking branch 'origin/SpringOne' into dixue/ai

dingmeng-xue committed:
update indentation

dingmeng-xue committed:
Update indentation

dingmeng-xue committed:
Remove lombok

Remove lombok

LGDoor committed:
fix: the duplicated data bug.

dingmeng-xue committed:
Add actuator

dingmeng-xue committed:
update

dingmeng-xue committed:
Update actuator settings

Update actuator settings

dingmeng-xue committed:
Move acme-assist to apps folder (#20)

dingmeng-xue committed:
Move python file to subfolder (#21)

dingmeng-xue committed:
Fix floating issue when price is greater than 1000 (#22)

* Move python file to subfolder

* Fix floating issue when it is greater than 1000

asaikali committed:
Merge branch 'Azure-Samples:Azure' into SpringOne

markpollack committed:
changed to use spring-ai 0.1.0-snapshot

asaikali committed:
Merge pull request #24 from markpollack/spring-ai-0.1.0-snapshot

Changed to use spring-ai 0.1.0-snapshot

dingmeng-xue committed:
Rebase SpringOne to Azure  (#26)

* -Initial Directory Setup.

* -Added Initial README.md

* Incremental Commit, separate TAP.

* -Adjusted Image Reference.

* -Incremental commit, folder adjusts.

* -Incremental commit, fix Architecture Image.

* -Incremental Commit. Refactored README.md files

* -Incremental Commit. Updated Doc.

* -Yanked depend-a-bot section from child pages.

* -Yanked depend-a-bot section from child pages.

* -Scripts folder take1

* updated stack to use jammy/base

* removed extra scripts/builder.json file

* updated az spring extension version and install steps

* updated ignored files and paths

* added output of env vars that are set

* updateded sections

* updated scripts and README.md

* updated README.md

* updated README.md

* updated README.md

* -Fix Unit1 anchor.

* -Fix Unit1 anchor.

* -Fix anchors.

* -Fixed main page.

* -Incremental commit.

* Delete .vscode directory

* -Yanked .vscode folder.

* -Added eme script.

* -Updated Doc

* -Updated Doc.

* -Yank X from template.sh files

* -Updated SCGW commands.

* -Updated SCGW commands.

* -Updated deploy commands

* -Updating SSO Steps.

* -Updated setup-sso-variables-ad.sh.

* -Yanked personal scriptfile.

* -Updated script permissons.

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated templates.

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated doc for IdentityService.

* -Updated doc for IdentityService.

* -Updated doc for IdentityService.

* -Updated doc for IdentityService.

* -Updated doc for PostGresDB.

* -Updated doc for PostGresDB.

* -Updated script templates.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated doc for RedisConnector.

* -Updated doc for RedisConnector.

* -Updated doc for Connectors.

* -Updated doc for Connectors.

* -Added Cumulative setup script. Doc Updates

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated GithubActions Workflows.

* -Updated cleanup.yaml

* -Updated GithubActionsDoc.

* -Added storage script template, instructions.

* -Updated ALL script.

* -Updated storage template script.

* -Updated GithubActions doc

* -Added storage script template, instructions.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Corrected WF name.

* -Fix provision.yml

* -Corrected pathing.

* updated provision.yml

* fixed path in provision.yml for gateway.sh

* update cart python env for GHA

* -Updated payment added BP_JVM flag

* updated python setup for GHA

* fix python setup for GHA

* added BP_JVM_VERSION to catalog.yml

* added BP_JVM_VERSION to identity.yml

* -Refactored apps to root.

* added BP_JVM_VERSION to order.yml

* removed BP_JP_VERSION order.yml

* removed env vars from order.yml

* -Updating deploy scripts

* -Updating deploy scripts

* Polish readme.md

* Polish Readme.md

* Polish readme.md

* Polish

* Polish readme.md

* Polish readme.md

* remove out of date diagarm

* Cleanup the docs for TAP

* Polish TAP install docs

* Add AI backend and sample data. (#1)

- Added a ChatGPT-like REST API which can chat with the user and answer the questions about the products.
- Added brand new bikes and accessories with detailed specs and images.

* Add 25 sample data and images for accessories. (#2)

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* Add frontend for AI (#5)

Co-authored-by: Zhe Li <lizhe@lizhe.org>
Co-authored-by: Zhe Li <zlhe@microsoft.com>
Co-authored-by: Zhe Li <Sneezry@users.noreply.github.com>

* Add pre-processing. (#6)

* Add code for preprocessing.

* Update the doc for preprocessing.

* add license for the images.

* Fix the data location.

---------

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* Update readme.md

* add ai into home and cart

* Tune the prompts.

* Rename the module and refactor code and API. (#8)

* merge: refactor code and API

update scripts

update greeting message

Update FAQ and images.

* Reformat and add comments.

---------

Co-authored-by: Dingmeng Xue <dixue@microsoft.com>
Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* update api

* update ui

* update ui

* bug fix

* bugfix

* update frontend

* Add logging for execution time.

* wording fix

* move ai chat to nav bar

* fix marked lib

* fix marked lib

* fix marked lib

* fix marked lib

* scroll top

* Change to gpt-35-turbo-16k and provide more context. (#15)

* Change to gpt-35-turbo-16k and provide more context.

* fix README.

---------

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* update faq and fix minor issue

* Remove lombok

* adding spring ai project, updates (#17)

* Some minor fixes.

* update indentation

* Update indentation

* fix: the duplicated data bug.

* Add actuator

* update

* Move acme-assist to apps folder (#20)

* Move python file to subfolder (#21)

* Fix floating issue when price is greater than 1000 (#22)

* Move python file to subfolder

* Fix floating issue when it is greater than 1000

* changed to use spring-ai 0.1.0-snapshot

* Fix path and do minor change after rebase

---------

Co-authored-by: ericm48 <me_manley@yahoo.com>
Co-authored-by: Anish Patel <anishs dot online dot bills at gmail.com>
Co-authored-by: asaikali <asaikali@gmail.com>
Co-authored-by: Raphael Yu <yyddxxmm@gmail.com>
Co-authored-by: Raphael Yu <xiangy@microsoft.com>
Co-authored-by: Zhe Li <lizhe@lizhe.org>
Co-authored-by: Zhe Li <zlhe@microsoft.com>
Co-authored-by: Zhe Li <Sneezry@users.noreply.github.com>
Co-authored-by: Mark Pollack <mpollack@vmware.com>

dingmeng-xue committed:
Merge branch 'Azure' into SpringOne

dingmeng-xue committed:
Remove duplicated files (#28)

* Remove duplicated files

* Remove duplicated file

asaikali committed:
Refactor package name to com.example

asaikali committed:
make the maven project id match the directory name

asaikali committed:
refactor package name to com.example

asaikali committed:
refactor package name to com.example

asaikali committed:
refactor package name to com.example

asaikali committed:
Add pyenv file

asaikali committed:
add tap catalog info

dingmeng-xue commented:
@asaikali , you can see other demo projects are using `com.vmware` as root of name space. Why do you need to change package name to `com.example`.  The whole project is under MIT. There is no license problem.

asaikali committed:
Add instructions on how to run the shopping cart service locally

asaikali committed:
update workload yaml

dingmeng-xue committed:
Correct path of apps (#31)

dingmeng-xue committed:
Include acme-assist AI app (#139)

* Add AI backend and sample data. (#1)

- Added a ChatGPT-like REST API which can chat with the user and answer the questions about the products.
- Added brand new bikes and accessories with detailed specs and images.

* Add 25 sample data and images for accessories. (#2)

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* Add frontend for AI (#5)

Co-authored-by: Zhe Li <lizhe@lizhe.org>
Co-authored-by: Zhe Li <zlhe@microsoft.com>
Co-authored-by: Zhe Li <Sneezry@users.noreply.github.com>

* Add pre-processing. (#6)

* Add code for preprocessing.

* Update the doc for preprocessing.

* add license for the images.

* Fix the data location.

---------

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* Update readme.md

* add ai into home and cart

* Tune the prompts.

* update api

* Rename the module and refactor code and API. (#8)

* merge: refactor code and API

update scripts

update greeting message

Update FAQ and images.

* Reformat and add comments.

---------

Co-authored-by: Dingmeng Xue <dixue@microsoft.com>
Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* update ui

* update ui

* bug fix

* bugfix

* update frontend

* Add logging for execution time.

* wording fix

* move ai chat to nav bar

* fix marked lib

* fix marked lib

* fix marked lib

* fix marked lib

* scroll top

* Change to gpt-35-turbo-16k and provide more context. (#15)

* Change to gpt-35-turbo-16k and provide more context.

* fix README.

---------

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* update faq and fix minor issue

* adding spring ai project, updates (#17)

* -Initial Directory Setup.

* -Added Initial README.md

* Incremental Commit, separate TAP.

* -Adjusted Image Reference.

* -Incremental commit, folder adjusts.

* -Incremental commit, fix Architecture Image.

* -Incremental Commit. Refactored README.md files

* -Incremental Commit. Updated Doc.

* -Yanked depend-a-bot section from child pages.

* -Yanked depend-a-bot section from child pages.

* -Scripts folder take1

* updated stack to use jammy/base

* removed extra scripts/builder.json file

* updated az spring extension version and install steps

* updated ignored files and paths

* added output of env vars that are set

* updateded sections

* updated scripts and README.md

* updated README.md

* updated README.md

* updated README.md

* -Fix Unit1 anchor.

* -Fix Unit1 anchor.

* -Fix anchors.

* -Fixed main page.

* -Incremental commit.

* Delete .vscode directory

* -Yanked .vscode folder.

* -Added eme script.

* -Updated Doc

* -Updated Doc.

* -Yank X from template.sh files

* -Updated SCGW commands.

* -Updated SCGW commands.

* -Updated deploy commands

* -Updating SSO Steps.

* -Updated setup-sso-variables-ad.sh.

* -Yanked personal scriptfile.

* -Updated script permissons.

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated templates.

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated doc for IdentityService.

* -Updated doc for IdentityService.

* -Updated doc for IdentityService.

* -Updated doc for IdentityService.

* -Updated doc for PostGresDB.

* -Updated doc for PostGresDB.

* -Updated script templates.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated doc for RedisConnector.

* -Updated doc for RedisConnector.

* -Updated doc for Connectors.

* -Updated doc for Connectors.

* -Added Cumulative setup script. Doc Updates

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated GithubActions Workflows.

* -Updated cleanup.yaml

* Some minor fixes.

* Remove lombok

* update indentation

* Update indentation

* fix: the duplicated data bug.

* Add actuator

* update

* Move acme-assist to apps folder (#20)

* Move python file to subfolder (#21)

* Fix floating issue when price is greater than 1000 (#22)

* Move python file to subfolder

* Fix floating issue when it is greater than 1000

* -Updated GithubActionsDoc.

* -Added storage script template, instructions.

* -Updated ALL script.

* -Updated storage template script.

* -Updated GithubActions doc

* -Added storage script template, instructions.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Corrected WF name.

* -Fix provision.yml

* -Corrected pathing.

* updated provision.yml

* fixed path in provision.yml for gateway.sh

* update cart python env for GHA

* -Updated payment added BP_JVM flag

* updated python setup for GHA

* fix python setup for GHA

* added BP_JVM_VERSION to catalog.yml

* added BP_JVM_VERSION to identity.yml

* -Refactored apps to root.

* added BP_JVM_VERSION to order.yml

* removed BP_JP_VERSION order.yml

* removed env vars from order.yml

* -Updating deploy scripts

* -Updating deploy scripts

* Polish readme.md

* Polish Readme.md

* Polish readme.md

* Polish

* Polish readme.md

* Polish readme.md

* remove out of date diagarm

* Cleanup the docs for TAP

* Polish TAP install docs

* changed to use spring-ai 0.1.0-snapshot

* Rebase SpringOne to Azure  (#26)

* -Initial Directory Setup.

* -Added Initial README.md

* Incremental Commit, separate TAP.

* -Adjusted Image Reference.

* -Incremental commit, folder adjusts.

* -Incremental commit, fix Architecture Image.

* -Incremental Commit. Refactored README.md files

* -Incremental Commit. Updated Doc.

* -Yanked depend-a-bot section from child pages.

* -Yanked depend-a-bot section from child pages.

* -Scripts folder take1

* updated stack to use jammy/base

* removed extra scripts/builder.json file

* updated az spring extension version and install steps

* updated ignored files and paths

* added output of env vars that are set

* updateded sections

* updated scripts and README.md

* updated README.md

* updated README.md

* updated README.md

* -Fix Unit1 anchor.

* -Fix Unit1 anchor.

* -Fix anchors.

* -Fixed main page.

* -Incremental commit.

* Delete .vscode directory

* -Yanked .vscode folder.

* -Added eme script.

* -Updated Doc

* -Updated Doc.

* -Yank X from template.sh files

* -Updated SCGW commands.

* -Updated SCGW commands.

* -Updated deploy commands

* -Updating SSO Steps.

* -Updated setup-sso-variables-ad.sh.

* -Yanked personal scriptfile.

* -Updated script permissons.

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated templates.

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated doc for IdentityService.

* -Updated doc for IdentityService.

* -Updated doc for IdentityService.

* -Updated doc for IdentityService.

* -Updated doc for PostGresDB.

* -Updated doc for PostGresDB.

* -Updated script templates.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated doc for RedisConnector.

* -Updated doc for RedisConnector.

* -Updated doc for Connectors.

* -Updated doc for Connectors.

* -Added Cumulative setup script. Doc Updates

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated GithubActions Workflows.

* -Updated cleanup.yaml

* -Updated GithubActionsDoc.

* -Added storage script template, instructions.

* -Updated ALL script.

* -Updated storage template script.

* -Updated GithubActions doc

* -Added storage script template, instructions.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Corrected WF name.

* -Fix provision.yml

* -Corrected pathing.

* updated provision.yml

* fixed path in provision.yml for gateway.sh

* update cart python env for GHA

* -Updated payment added BP_JVM flag

* updated python setup for GHA

* fix python setup for GHA

* added BP_JVM_VERSION to catalog.yml

* added BP_JVM_VERSION to identity.yml

* -Refactored apps to root.

* added BP_JVM_VERSION to order.yml

* removed BP_JP_VERSION order.yml

* removed env vars from order.yml

* -Updating deploy scripts

* -Updating deploy scripts

* Polish readme.md

* Polish Readme.md

* Polish readme.md

* Polish

* Polish readme.md

* Polish readme.md

* remove out of date diagarm

* Cleanup the docs for TAP

* Polish TAP install docs

* Add AI backend and sample data. (#1)

- Added a ChatGPT-like REST API which can chat with the user and answer the questions about the products.
- Added brand new bikes and accessories with detailed specs and images.

* Add 25 sample data and images for accessories. (#2)

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* Add frontend for AI (#5)

Co-authored-by: Zhe Li <lizhe@lizhe.org>
Co-authored-by: Zhe Li <zlhe@microsoft.com>
Co-authored-by: Zhe Li <Sneezry@users.noreply.github.com>

* Add pre-processing. (#6)

* Add code for preprocessing.

* Update the doc for preprocessing.

* add license for the images.

* Fix the data location.

---------

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* Update readme.md

* add ai into home and cart

* Tune the prompts.

* Rename the module and refactor code and API. (#8)

* merge: refactor code and API

update scripts

update greeting message

Update FAQ and images.

* Reformat and add comments.

---------

Co-authored-by: Dingmeng Xue <dixue@microsoft.com>
Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* update api

* update ui

* update ui

* bug fix

* bugfix

* update frontend

* Add logging for execution time.

* wording fix

* move ai chat to nav bar

* fix marked lib

* fix marked lib

* fix marked lib

* fix marked lib

* scroll top

* Change to gpt-35-turbo-16k and provide more context. (#15)

* Change to gpt-35-turbo-16k and provide more context.

* fix README.

---------

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* update faq and fix minor issue

* Remove lombok

* adding spring ai project, updates (#17)

* Some minor fixes.

* update indentation

* Update indentation

* fix: the duplicated data bug.

* Add actuator

* update

* Move acme-assist to apps folder (#20)

* Move python file to subfolder (#21)

* Fix floating issue when price is greater than 1000 (#22)

* Move python file to subfolder

* Fix floating issue when it is greater than 1000

* changed to use spring-ai 0.1.0-snapshot

* Fix path and do minor change after rebase

---------

Co-authored-by: ericm48 <me_manley@yahoo.com>
Co-authored-by: Anish Patel <anishs dot online dot bills at gmail.com>
Co-authored-by: asaikali <asaikali@gmail.com>
Co-authored-by: Raphael Yu <yyddxxmm@gmail.com>
Co-authored-by: Raphael Yu <xiangy@microsoft.com>
Co-authored-by: Zhe Li <lizhe@lizhe.org>
Co-authored-by: Zhe Li <zlhe@microsoft.com>
Co-authored-by: Zhe Li <Sneezry@users.noreply.github.com>
Co-authored-by: Mark Pollack <mpollack@vmware.com>

* Remove duplicated files (#28)

* Remove duplicated files

* Remove duplicated file

* Refactor package name to com.example

* make the maven project id match the directory name

* refactor package name to com.example

* refactor package name to com.example

* refactor package name to com.example

* Add pyenv file

* add tap catalog info

* Add instructions on how to run the shopping cart service locally

* update workload yaml

* Correct path of apps (#31)

---------

Co-authored-by: Raphael Yu <yyddxxmm@gmail.com>
Co-authored-by: Adib Saikali <asaikali@users.noreply.github.com>
Co-authored-by: Raphael Yu <xiangy@microsoft.com>
Co-authored-by: Zhe Li <lizhe@lizhe.org>
Co-authored-by: Zhe Li <zlhe@microsoft.com>
Co-authored-by: Zhe Li <Sneezry@users.noreply.github.com>
Co-authored-by: Mark Pollack <mpollack@vmware.com>
Co-authored-by: ericm48 <me_manley@yahoo.com>
Co-authored-by: Anish Patel <anishs dot online dot bills at gmail.com>
Co-authored-by: asaikali <asaikali@gmail.com>

asaikali committed:
fix package names

asaikali committed:
fix package names and typos

nevenc committed:
SpringOne  (#32)

* Updated catalog app with new images.

* Added Azure OpenAI integration.

dingmeng-xue committed:
Merge remote-tracking branch 'upstream/Azure' into SpringOne

dingmeng-xue committed:
Updated architecture diagram and static content (#141)

* Add AI backend and sample data. (#1)

- Added a ChatGPT-like REST API which can chat with the user and answer the questions about the products.
- Added brand new bikes and accessories with detailed specs and images.

* Add 25 sample data and images for accessories. (#2)

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* Add frontend for AI (#5)

Co-authored-by: Zhe Li <lizhe@lizhe.org>
Co-authored-by: Zhe Li <zlhe@microsoft.com>
Co-authored-by: Zhe Li <Sneezry@users.noreply.github.com>

* Add pre-processing. (#6)

* Add code for preprocessing.

* Update the doc for preprocessing.

* add license for the images.

* Fix the data location.

---------

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* Update readme.md

* add ai into home and cart

* Tune the prompts.

* update api

* Rename the module and refactor code and API. (#8)

* merge: refactor code and API

update scripts

update greeting message

Update FAQ and images.

* Reformat and add comments.

---------

Co-authored-by: Dingmeng Xue <dixue@microsoft.com>
Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* update ui

* update ui

* bug fix

* bugfix

* update frontend

* Add logging for execution time.

* wording fix

* move ai chat to nav bar

* fix marked lib

* fix marked lib

* fix marked lib

* fix marked lib

* scroll top

* Change to gpt-35-turbo-16k and provide more context. (#15)

* Change to gpt-35-turbo-16k and provide more context.

* fix README.

---------

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* update faq and fix minor issue

* adding spring ai project, updates (#17)

* -Initial Directory Setup.

* -Added Initial README.md

* Incremental Commit, separate TAP.

* -Adjusted Image Reference.

* -Incremental commit, folder adjusts.

* -Incremental commit, fix Architecture Image.

* -Incremental Commit. Refactored README.md files

* -Incremental Commit. Updated Doc.

* -Yanked depend-a-bot section from child pages.

* -Yanked depend-a-bot section from child pages.

* -Scripts folder take1

* updated stack to use jammy/base

* removed extra scripts/builder.json file

* updated az spring extension version and install steps

* updated ignored files and paths

* added output of env vars that are set

* updateded sections

* updated scripts and README.md

* updated README.md

* updated README.md

* updated README.md

* -Fix Unit1 anchor.

* -Fix Unit1 anchor.

* -Fix anchors.

* -Fixed main page.

* -Incremental commit.

* Delete .vscode directory

* -Yanked .vscode folder.

* -Added eme script.

* -Updated Doc

* -Updated Doc.

* -Yank X from template.sh files

* -Updated SCGW commands.

* -Updated SCGW commands.

* -Updated deploy commands

* -Updating SSO Steps.

* -Updated setup-sso-variables-ad.sh.

* -Yanked personal scriptfile.

* -Updated script permissons.

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated templates.

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated doc for IdentityService.

* -Updated doc for IdentityService.

* -Updated doc for IdentityService.

* -Updated doc for IdentityService.

* -Updated doc for PostGresDB.

* -Updated doc for PostGresDB.

* -Updated script templates.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated doc for RedisConnector.

* -Updated doc for RedisConnector.

* -Updated doc for Connectors.

* -Updated doc for Connectors.

* -Added Cumulative setup script. Doc Updates

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated GithubActions Workflows.

* -Updated cleanup.yaml

* Some minor fixes.

* Remove lombok

* update indentation

* Update indentation

* fix: the duplicated data bug.

* Add actuator

* update

* Move acme-assist to apps folder (#20)

* Move python file to subfolder (#21)

* Fix floating issue when price is greater than 1000 (#22)

* Move python file to subfolder

* Fix floating issue when it is greater than 1000

* -Updated GithubActionsDoc.

* -Added storage script template, instructions.

* -Updated ALL script.

* -Updated storage template script.

* -Updated GithubActions doc

* -Added storage script template, instructions.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Corrected WF name.

* -Fix provision.yml

* -Corrected pathing.

* updated provision.yml

* fixed path in provision.yml for gateway.sh

* update cart python env for GHA

* -Updated payment added BP_JVM flag

* updated python setup for GHA

* fix python setup for GHA

* added BP_JVM_VERSION to catalog.yml

* added BP_JVM_VERSION to identity.yml

* -Refactored apps to root.

* added BP_JVM_VERSION to order.yml

* removed BP_JP_VERSION order.yml

* removed env vars from order.yml

* -Updating deploy scripts

* -Updating deploy scripts

* Polish readme.md

* Polish Readme.md

* Polish readme.md

* Polish

* Polish readme.md

* Polish readme.md

* remove out of date diagarm

* Cleanup the docs for TAP

* Polish TAP install docs

* changed to use spring-ai 0.1.0-snapshot

* Rebase SpringOne to Azure  (#26)

* -Initial Directory Setup.

* -Added Initial README.md

* Incremental Commit, separate TAP.

* -Adjusted Image Reference.

* -Incremental commit, folder adjusts.

* -Incremental commit, fix Architecture Image.

* -Incremental Commit. Refactored README.md files

* -Incremental Commit. Updated Doc.

* -Yanked depend-a-bot section from child pages.

* -Yanked depend-a-bot section from child pages.

* -Scripts folder take1

* updated stack to use jammy/base

* removed extra scripts/builder.json file

* updated az spring extension version and install steps

* updated ignored files and paths

* added output of env vars that are set

* updateded sections

* updated scripts and README.md

* updated README.md

* updated README.md

* updated README.md

* -Fix Unit1 anchor.

* -Fix Unit1 anchor.

* -Fix anchors.

* -Fixed main page.

* -Incremental commit.

* Delete .vscode directory

* -Yanked .vscode folder.

* -Added eme script.

* -Updated Doc

* -Updated Doc.

* -Yank X from template.sh files

* -Updated SCGW commands.

* -Updated SCGW commands.

* -Updated deploy commands

* -Updating SSO Steps.

* -Updated setup-sso-variables-ad.sh.

* -Yanked personal scriptfile.

* -Updated script permissons.

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated templates.

* -Updated README.md

* -Updated README.md

* -Updated README.md

* -Updated doc for IdentityService.

* -Updated doc for IdentityService.

* -Updated doc for IdentityService.

* -Updated doc for IdentityService.

* -Updated doc for PostGresDB.

* -Updated doc for PostGresDB.

* -Updated script templates.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated PG setup doc.

* -Updated doc for RedisConnector.

* -Updated doc for RedisConnector.

* -Updated doc for Connectors.

* -Updated doc for Connectors.

* -Added Cumulative setup script. Doc Updates

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated Doc.

* -Updated GithubActions Workflows.

* -Updated cleanup.yaml

* -Updated GithubActionsDoc.

* -Added storage script template, instructions.

* -Updated ALL script.

* -Updated storage template script.

* -Updated GithubActions doc

* -Added storage script template, instructions.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Updated GitHubActions doc.

* -Corrected WF name.

* -Fix provision.yml

* -Corrected pathing.

* updated provision.yml

* fixed path in provision.yml for gateway.sh

* update cart python env for GHA

* -Updated payment added BP_JVM flag

* updated python setup for GHA

* fix python setup for GHA

* added BP_JVM_VERSION to catalog.yml

* added BP_JVM_VERSION to identity.yml

* -Refactored apps to root.

* added BP_JVM_VERSION to order.yml

* removed BP_JP_VERSION order.yml

* removed env vars from order.yml

* -Updating deploy scripts

* -Updating deploy scripts

* Polish readme.md

* Polish Readme.md

* Polish readme.md

* Polish

* Polish readme.md

* Polish readme.md

* remove out of date diagarm

* Cleanup the docs for TAP

* Polish TAP install docs

* Add AI backend and sample data. (#1)

- Added a ChatGPT-like REST API which can chat with the user and answer the questions about the products.
- Added brand new bikes and accessories with detailed specs and images.

* Add 25 sample data and images for accessories. (#2)

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* Add frontend for AI (#5)

Co-authored-by: Zhe Li <lizhe@lizhe.org>
Co-authored-by: Zhe Li <zlhe@microsoft.com>
Co-authored-by: Zhe Li <Sneezry@users.noreply.github.com>

* Add pre-processing. (#6)

* Add code for preprocessing.

* Update the doc for preprocessing.

* add license for the images.

* Fix the data location.

---------

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* Update readme.md

* add ai into home and cart

* Tune the prompts.

* Rename the module and refactor code and API. (#8)

* merge: refactor code and API

update scripts

update greeting message

Update FAQ and images.

* Reformat and add comments.

---------

Co-authored-by: Dingmeng Xue <dixue@microsoft.com>
Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* update api

* update ui

* update ui

* bug fix

* bugfix

* update frontend

* Add logging for execution time.

* wording fix

* move ai chat to nav bar

* fix marked lib

* fix marked lib

* fix marked lib

* fix marked lib

* scroll top

* Change to gpt-35-turbo-16k and provide more context. (#15)

* Change to gpt-35-turbo-16k and provide more context.

* fix README.

---------

Co-authored-by: Raphael Yu <xiangy@microsoft.com>

* update faq and fix minor issue

* Remove lombok

* adding spring ai project, updates (#17)

* Some minor fixes.

* update indentation

* Update indentation

* fix: the duplicated data bug.

* Add actuator

* update

* Move acme-assist to apps folder (#20)

* Move python file to subfolder (#21)

* Fix floating issue when price is greater than 1000 (#22)

* Move python file to subfolder

* Fix floating issue when it is greater than 1000

* changed to use spring-ai 0.1.0-snapshot

* Fix path and do minor change after rebase

---------

Co-authored-by: ericm48 <me_manley@yahoo.com>
Co-authored-by: Anish Patel <anishs dot online dot bills at gmail.com>
Co-authored-by: asaikali <asaikali@gmail.com>
Co-authored-by: Raphael Yu <yyddxxmm@gmail.com>
Co-authored-by: Raphael Yu <xiangy@microsoft.com>
Co-authored-by: Zhe Li <lizhe@lizhe.org>
Co-authored-by: Zhe Li <zlhe@microsoft.com>
Co-authored-by: Zhe Li <Sneezry@users.noreply.github.com>
Co-authored-by: Mark Pollack <mpollack@vmware.com>

* Remove duplicated files (#28)

* Remove duplicated files

* Remove duplicated file

* Refactor package name to com.example

* make the maven project id match the directory name

* refactor package name to com.example

* refactor package name to com.example

* refactor package name to com.example

* Add pyenv file

* add tap catalog info

* Add instructions on how to run the shopping cart service locally

* update workload yaml

* Correct path of apps (#31)

* fix package names

* fix package names and typos

* SpringOne  (#32)

* Updated catalog app with new images.

* Added Azure OpenAI integration.

---------

Co-authored-by: Raphael Yu <yyddxxmm@gmail.com>
Co-authored-by: Adib Saikali <asaikali@users.noreply.github.com>
Co-authored-by: Raphael Yu <xiangy@microsoft.com>
Co-authored-by: Zhe Li <lizhe@lizhe.org>
Co-authored-by: Zhe Li <zlhe@microsoft.com>
Co-authored-by: Zhe Li <Sneezry@users.noreply.github.com>
Co-authored-by: Mark Pollack <mpollack@vmware.com>
Co-authored-by: ericm48 <me_manley@yahoo.com>
Co-authored-by: Anish Patel <anishs dot online dot bills at gmail.com>
Co-authored-by: asaikali <asaikali@gmail.com>
Co-authored-by: Neven C <neven.cvetkovic@gmail.com>

dingmeng-xue committed:
Merge remote-tracking branch 'upstream/Azure' into SpringOne

nevenc committed:
SpringOne updates (#34)

* Updated catalog app with new images.

* Added Azure OpenAI integration.

* Updated with screenshots for AI services.

nevenc committed:
SpringOne - syncing AI updates. (#35)

* Updated catalog app with new images.

* Added Azure OpenAI integration.

* Updated with screenshots for AI services.

* Fixed typo - image of "Verden MIK Cargo Trunk Bag" pointing to "new_accessories_1.jpg"

nevenc committed:
Added AI workshop instructions.

asaikali committed:
Merge pull request #142 from selvasingh/SpringOne

Spring one

LGDoor committed:
Update README.md
